statsd/test_helpers_test.go

```
@@ -0,0 +1,652 @@
+	ts.conn.Close()
+	close(ts.stopped)
+}
+
+func (ts *testServer) wait(t *testing.T, nbExpectedMetric int, timeout int, waitForTelemetry bool) {
+	start := time.Now()
+	for {
+		ts.Lock()
+		if nbExpectedMetric <= len(ts.data) {
+			ts.Unlock()
+			return
+		} else if time.Now().Sub(start) > time.Duration(timeout)*time.Second {
```

golangci-lint (1.62.2)

S1012: should use `time.Since` instead of `time.Now().Sub`

(gosimple)